script/tasks/diamondUpdateFacet.sh

```
@@ -143,7 +143,7 @@ diamondUpdateFacet() {
         if [ "$FACET_CUT" != "0x" ]; then
           echo "Proposing facet cut for $script on network $NETWROK..."
           DIAMOND_ADDRESS=$(getContractAddressFromDeploymentLogs "$NETWORK" "$ENVIRONMENT" "$DIAMOND_CONTRACT_NAME")
-          ts-node script/deploy/safe/propose-to-safe.ts --to "$DIAMOND_ADDRESS" --calldata "$FACET_CUT" --network "$NETWORK" --rpcUrl $(getRPCUrl $NETWORK) --privateKey "$SAFE_SIGNER_PRIVATE_KEY"
+          npx tsx script/deploy/safe/propose-to-safe.ts --to "$DIAMOND_ADDRESS" --calldata "$FACET_CUT" --network "$NETWORK" --rpcUrl $(getRPCUrl $NETWORK) --privateKey "$SAFE_SIGNER_PRIVATE_KEY"
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)